src/middlewared/middlewared/plugins/pool.py

```diff
@@ -476,6 +477,7 @@ def pool_extend(self, pool):
                 'topology': None,
                 'healthy': False,
                 'status_detail': None,
+                'autotrim': {},
```

Comment:
Why `{}`?, isn't it on/off?

Reply:
@themylogin no, its an dict like other properties: {"parsed": "XXX", "value": "XX", "inherit": "XXX"}

Reply:
But what does `{}` mean then? Does py-libzfs return values like this? IMO this will be erroneously used like `pool["autotrim"]["value"]` and it does not matter if we crash on `KeyError("autotrim")` or `KeyError("value")`